static/dbstructure.config.php

```diff
@@ -1363,11 +1363,11 @@
 	"post-engagement" => [
 		"comment" => "Engagement data per post",
 		"fields" => [
-			"uri-id" => ["type" => "int unsigned", "not null" => "1", "primary" => "1",  "foreign" => ["item-uri" => "id"], "comment" => "Id of the item-uri table entry that contains the item uri"],
+			"uri-id" => ["type" => "int unsigned", "not null" => "1", "primary" => "1", "foreign" => ["item-uri" => "id"], "comment" => "Id of the item-uri table entry that contains the item uri"],
 			"owner-id" => ["type" => "int unsigned", "not null" => "1", "default" => "0", "foreign" => ["contact" => "id"], "comment" => "Item owner"],
 			"contact-type" => ["type" => "tinyint", "not null" => "1", "default" => "0", "comment" => "Person, organisation, news, community, relay"],
 			"media-type" => ["type" => "tinyint", "not null" => "1", "default" => "0", "comment" => "Type of media in a bit array (1 = image, 2 = video, 4 = audio"],
-			"language" => ["type" => "varchar(128)", "comment" => "Language information about this post"],
+			"iso-639-1" => ["type" => "char(2)", "comment" => "Language information about this post in the ISO 639-1 format"],
```

Comment:
Since the field can now host only one language code, we should say it's the post "primary language". Also, is this language automatically detected or extracted from the post metadata?

Reply:
It depends. AFAIK it can happen that the detection doesn't find a language. Then this will be the one the one that was provided in the post.

Reply:
I renamed the field.